crates/era-cheatcodes/src/cheatcodes.rs

```diff
@@ -188,6 +189,12 @@ enum FinishCycleOneTimeActions {
     RevokePersistentAccounts { accounts: Vec<H160> },
 }
 
+#[derive(Debug, Clone)]
+struct ForceRevertError {
+    abi_encode: bool,
```

Comment:
We could remove this and use a constructor `ForceRevertError::abi_encode(data)` vs `ForceRevertError::raw(bytes)`, with raw padding as necessary

Reply:
Improved! Thanks for the suggestion